lucien-ios/Dashboard/LendingCollectionViewController.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  LendingViewController.swift
+//  lucien-ios
+//
+//  Created by Fang, Gracie on 10/9/17.
+//  Copyright © 2017 Intrepid Pursuits. All rights reserved.
+//
+
+import UIKit
+
+final class LendingCollectionViewController: UIViewController, UICollectionViewDataSource, UICollectionViewDelegate, UICollectionViewDelegateFlowLayout {
+
+    @IBOutlet weak var collectionView: UICollectionView!
+
+    let reuseIdentifier = "comicCell"
+    let objects = ["Princess", "Gracie", "Kittens"]
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.collectionView.register(UINib(nibName:"LendingCollectionViewCell", bundle: nil), forCellWithReuseIdentifier: reuseIdentifier)
+        self.collectionView.delegate = self
+        self.collectionView.dataSource = self
+        setLayout()
+    }
+
+    func setLayout() {
+        let collectionViewLayout = collectionView.collectionViewLayout as? UICollectionViewFlowLayout
+        let sectionInset = UIEdgeInsetsMake(0.0, 24.0, 0.0, 0.0)
+        collectionViewLayout?.sectionInset = sectionInset
+    }
+
+    func numberOfSections(in collectionView: UICollectionView) -> Int { return 1 }
```

Comment:
Function definitions should never be one-liners. Also, 1 is the default value returned so you don't actually need to implement this